sample-apps/radiology/lib/transforms/transforms.py

```
@@ -27,6 +38,163 @@
+                raise ValueError(f"Expected a directory path for key '{key}', got: {dir_path}")
+
+            # Gather all files in directory
+            image_files = sorted(
+                [
+                    os.path.join(dir_path, f)
+                    for f in os.listdir(dir_path)
+                    if f.lower().endswith((".nii", ".nii.gz", ".nrrd"))
+                ]
+            )
+            if not image_files:
+                raise FileNotFoundError(f"No NIfTI images found in directory {dir_path}")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)